src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Expected/Binskim.linux-x64.dll.sarif

```diff
@@ -772,10 +772,10 @@
           "rules": [
             {
               "id": "BA2001",
+              "name": "LoadImageAboveFourGigabyteAddress",
```

Comment:
>name [](http://example.com/codeflow?start=15&length=4)

with the sarif-sdk change, we are enforcing a specific order where name will be close to id and helpuri will be near the end.

Reviewed all files and only two has `unexpected` changes that are commented as well.